src/components/ui4/CardUi4.tsx

```diff
@@ -1,44 +1,56 @@
 import * as React from 'react'
-import { StyleSheet, TouchableHighlight, TouchableOpacity, View } from 'react-native'
+import { StyleSheet, TouchableOpacity, View } from 'react-native'
 import LinearGradient, { LinearGradientProps } from 'react-native-linear-gradient'
 import AntDesignIcon from 'react-native-vector-icons/AntDesign'
 
 import { useHandler } from '../../hooks/useHandler'
 import { triggerHaptic } from '../../util/haptic'
+import { fixSides, mapSides, sidesToMargin } from '../../util/sides'
 import { showError } from '../services/AirshipInstance'
 import { cacheStyles, Theme, useTheme } from '../services/ThemeContext'
 import { SectionView } from './SectionView'
 
-export type CardType = 'default' | 'warning' | 'error'
-
 interface Props {
-  children: React.ReactNode | React.ReactNode[] // Top layer
+  // Top layer:
+  overlay?: React.ReactNode // Rendered above/on top of children
 
+  // children & icon share the same 2nd layer:
+  children: React.ReactNode | React.ReactNode[]
   icon?: React.ReactNode
-  overlay?: React.ReactNode
 
-  // These two override the default theme background:
-  underlayForeground?: React.ReactNode // Middle layer, e.g. embedded images as part of the background
-  underlayBackground?: LinearGradientProps // Bottom layer gradient
+  marginRem?: number[] | number
+
+  underlayForeground?: React.ReactNode // 3rd layer, e.g. embedded images as part of the background
+  underlayBackground?: LinearGradientProps // Bottom-most layer
+
+  // Options:
+  sections?: Boolean // Automatic section dividers, only if chilren is multiple nodes
```

Comment:
Change this to lowercase `boolean`. The uppercase boolean is something different.